python/src/sdf/_ignition_sdformat_pybind11.cc

```diff
@@ -16,6 +16,10 @@
 
 #include <pybind11/pybind11.h>
 
+#include "Error.hh"
```

Comment:
Naming these `.cc` and `.hh` files the same as the ones in the root `src/` and `include/sdf` seems like it could lead to confusion. Should we prefix/suffix them with `py` (eg. `pyError.hh` or `Error_py.hh`)? /cc @scpeters

Reply:
we didn't do that for ign-math, but I see your point

https://github.com/ignitionrobotics/ign-math/tree/ign-math6/src/python_pybind11/src